AxonDeepSeg/segment.py

```diff
@@ -181,17 +199,15 @@ def main(argv=None):
     requiredName = ap.add_argument_group('required arguments')
 
     # Setting the arguments of the segmentation
-    requiredName.add_argument('-t', '--type', required=True, choices=['SEM','TEM', 'OM'], help='Type of acquisition to segment. \n'+
+    requiredName.add_argument('-t', '--type', required=True, choices=['SEM', 'TEM', 'BF'], help='Type of acquisition to segment. \n'+
                                                                                         'SEM: scanning electron microscopy samples. \n'+
-                                                                                        'TEM: transmission electron microscopy samples. \n'+
-                                                                                        'OM: optical microscopy samples')
+                                                                                        'TEM: transmission electron microscopy samples. \n' +
+                                                                                        'BF: Bright-field microscopy samples.')
     requiredName.add_argument('-i', '--imgpath', required=True, nargs='+', help='Path to the image to segment or path to the folder \n'+
                                                                                 'where the image(s) to segment is/are located.')
 
     ap.add_argument("-m", "--model", required=False, help='Folder where the model is located. \n'+
-                                                          'The default SEM model path is: \n'+str(default_SEM_path)+'\n'+
-                                                          'The default TEM model path is: \n'+str(default_TEM_path)+'\n'+
-                                                          'The default OM model path is: \n'+str(model_seg_pns_bf_path)+'\n')
+                                                          'The default SEM model path is: \n'+str(default_SEM_path))
```

Comment:
I don't understand the `-m` argument, we already have `-t` above for the model. Is this really needed?
I tried with `-m` and without `-t` and it fails as `-t` is required. Am I missing something?
And why only SEM in the help?

Reply:
I think -m is for when we have custom models, i.e. non-default ones. You can provide the folder path to it.

Reply:
Ok thanks, I get it now.
So we have to define `-t` with SEM, TEM or BF, and if `-m` is not specified the default model will be used.
That make sense for our new v4 because the "model resolution" is in the model config file itself.

What had me confused if that it makes little sense for our v3 because the "model resolution" was hardcoded for each modality. So we could use a different model but not specify a different model resolution.

In any case, I would remove the line 210 here. and in 209 say: `Folder where the model is located, if different from AxonDeepSeg default models.`

Reply:
Resolved in fdcab748cf